policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl

```diff
@@ -6,14 +6,19 @@
     strategy:
       fail-fast: false
       matrix:
-  {{- if and .Branchvals.Cgo (has "el7"  .Branchvals.Features) }}{{/* release-3.x version uses the same image for both el7 and non el7 */}}
-        golang_cross: [ {{ .Branchvals.GoVersion }}, {{ .Branchvals.GoVersion }}-el7 ]
+  {{- if and .Branchvals.Cgo (has "el7"  .Branchvals.Features) }}
+    {{- if eq .Branch "release-5-lts" }}
+        golang_cross: [ {{ .Branchvals.GoVersion }}, 1.16-el7 ]
+    {{- end }}
```

Comment:
I think the same if...else if... construction used below should be used here too.